packages/api/internal/handlers/sandboxes_list.go

```diff
@@ -85,5 +86,10 @@ func (a *APIStore) GetSandboxes(c *gin.Context) {
 		sandboxes = append(sandboxes, instance)
 	}
 
+	// Sort sandboxes by start time descending
+	sort.Slice(sandboxes, func(i, j int) bool {
+		return sandboxes[i].StartedAt.After(sandboxes[j].StartedAt)
```

Comment:
It returns list sorted in ascending order. The newest sandbox will always "win" and it will be the first in the slice. 
```suggestion
		return sandboxes[i].StartedAt.Before(sandboxes[j].StartedAt)
```